pre-commit

```diff
@@ -19,6 +19,14 @@ if [ ! -e "$DEV_LIB_PATH/check-diff.sh" ]; then
 	exit 1
 fi
 
+DEV_LIB_COMMIT_MESSAGE_FILE=".dev-lib-commit-message.$PPID"
+DEV_LIB_COMMIT_MESSAGE=''
+function add_commit_message {
+	label=$1
+	value=$2
+	DEV_LIB_COMMIT_MESSAGE+="\n$label: $value"
```

Comment:
Good point, but I don't think that's so important now, since the file is getting stored in `/tmp`. It will automatically get garbage-collected with the next reboot.